value-fixture/src/org/immutables/fixture/jdkonly/UsingAllOptionals.java

```diff
@@ -59,6 +62,7 @@ void use() {
               .d1(1.1)
               .l1(OptionalLong.empty())
               .jso(javaslang.control.Option.none())
+              .vo(io.vavr.control.Option.none())
```

Comment:
just to note: even if there are precedent with some other variants above, this test is actually testing nothing in that sense that if you remove support for the javaslang.Option you've just added, that accessor will still be generated as this is the default to just have an initializer with the same type as return type. The cases for `v1`, `v2`, and `d1` actually show specially generated initializer like `Builder i1(int int)` for `OptionalInt i1()`, but `Builder i1(OptionalInt i)` is not interesting as it would be generated regardless. So that's the trick

Reply:
It's good to know, thank you